ia/src/ml_agent.py

```
@@ -49,7 +47,16 @@ def get_action(client, needed_resources):
   closest_resources = search_closest_resource(client, needed_resources)
 
   if closest_resources is None:
-    return utils.RIGHT, None
+    client["move"]["consecutive_turns"] += 1
+    if client["move"]["consecutive_turns"] >= 3:
+            client["move"]["consecutive_turns"] = 0
+            client["move"]["forward"] = True
+            return utils.FORWARD, None
+    else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)